finagle-netty4/src/test/scala/com/twitter/finagle/netty4/ssl/server/Netty4ServerEngineFactoryTest.scala

```diff
@@ -112,6 +116,26 @@ class Netty4ServerEngineFactoryTest extends FunSuite {
     }
   }
 
+  test("config with KeyManager succeeds") {
+    // ToDo: Should we also create tests / build the functionality that the keymanagers are rejected if containing expired certs
```

Comment:
how hard is this? 

i defer to @ryanoneill

Reply:
I have not added the test. Depending on how the KeyManagerFactory is constructed the check is already performed there.